package.json

```diff
@@ -10,8 +10,9 @@
   "scripts": {
     "build": "babel src --out-dir lib --ignore test.js,test.jsx --source-maps",
     "coveralls": "cat coverage/lcov.info | coveralls",
+    "flow": "flow",
     "lint": "eslint . --ext .js,.jsx --fix",
-    "test": "yarn run test:unit && yarn run test:integration",
+    "test": "yarn run flow && yarn run test:unit && yarn run test:integration",
```

Comment:
will keep an eye out for this, thanks. Since this package is built much differently in Travis than the platform apps (no Docker, using more built-in Travis APIs), it may end up not being an issue for us but I'll let you know.